src/schema/v2/schema.ts

```diff
@@ -101,6 +102,7 @@ const rootFields = {
   artwork: Artwork,
   // artworkVersion: ArtworkVersionResolver,
   artworksConnection: filterArtworksConnection(),
+  artworks: Artworks,
```

Comment:
can prefix this with `_do_not_use_artworks: Artworks` to be even stronger than the deprecation reason.